core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -187,6 +192,7 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
                   s"userNamespace: ${r.msg.user.namespace.name}, action: ${r.action}, " +
                   s"needed memory: ${r.action.limits.memory.megabytes} MB, " +
                   s"waiting messages: ${runBuffer.size}")(r.msg.transid)
+              MetricEmitter.emitCounterMetric(LoggingMarkers.CONTAINER_POOL_RESCHEDULED_ACTIVATION)
```

Comment:
Note that this will currently only emit once-per-log-timeout - so it will not be accurate; once #4593 is merged, this metric should be counted every message loop (since message processing should no longer be in a "tight" loop).